internal/pkg/cli/config/config.go

```diff
@@ -88,6 +88,14 @@ func Parse(args []string) (Config, error) {
 		return cfg, fmt.Errorf("cannot parse flags: %w", err)
 	}
 
+	// Define mapstructure hooks
+	hooks := viper.DecodeHook(
+		mapstructure.ComposeDecodeHookFunc(
+			mapstructure.TextUnmarshallerHookFunc(),
+			mapstructure.StringToTimeDurationHookFunc(),
```

Comment:
Enabled support for `time.Duration` type by `StringToTimeDurationHookFunc`.

Reply:
I'll check the docs today and ask you on Monday. 🙂

Reply:
ok